src/foam/parse/QueryParser.js

```diff
@@ -106,8 +106,7 @@ foam.CLASS({
             sym('dot'),
             sym('equals'),
             sym('before'),
-            sym('after'),
-            sym('id')
```

Comment:
@kgrgreer why was it here

Reply:
It is for searching by primary key. But it looks like it only works for integer ids as it is currently implemented.